### PR TITLE
Use python imports to identify fixtures (part 2/6)

### PR DESCRIPTION
**Pull request by cognifloyd**
### Background

I'm working towards introducing [`pants`](https://www.pantsbuild.org/docs). Eventually I would like to use pants to run tests to take advantage of the fine-grained per-file caching of results that accounts for dependencies by python files (pants infers the deps by reading the python files).

In order to use the fine-grained caching, Pants needs to know which tests rely on which fixtures. We could add **extra** metadata to tie the tests to the fixtures, but that would be an additional maintenance burden that will become out-of-date. We can also include all fixtures for all tests, but then we don't benefit from the fine-grained per-file caching. However, pants can already infer dependencies based on python imports, so that is what this PR (and several follow-up PRs) takes advantage of.

### Overview

This PR builds on, and follows the patterns established in, #5699.

This PR does the following:
- turn every fixture into an importable python module with an `__init__.py`.
- add a `fixture.py` file in each fixture that uses the fixturesloader utils (where helpful) to identify itself with `PATH` and `NAME` vars.
- in every test that uses a given fixture, import `PATH` and/or `NAME` vars from that fixture.

This PR updates all fixture packs, skipping any fixtures that need something slightly different (like `dummy_pack_7` and `dummy_pack_9`).
Follow-up PRs will address other fixture packs and other sets of fixtures. I will submit those PRs after this one is merged.

### CODE REVIEW

**Comment by cognifloyd** [comment]
Stats for this PR:

**32** pack fixtures got 62 new files (a `fixture.py` in each; a `__init__.py` in all but 2 pack fixtures which already had it)

All 32 copies of `fixture.py` have the same 16 lines, 13 of which are our copyright/license header.

The remaining 38 files (=100-62) are test files that use the vars from the pack fixtures.
Lines changed in test files:
- `-298` lines
- `+238` lines _= 750 lines - 512 lines (512 = 16 copied lines * 32 `fixture.py` files)_

File: contrib/core/tests/test_action_sendmail.py [hunk 1]
```diff
@@ -38,6 +39,10 @@
 MOCK_EXECUTION.id = "598dbf0c0640fd54bffc688b"
```
Comment (rush-skills) [inline contrib/core/tests/test_action_sendmail.py:39]: Not relevant to your changes, but do we need this hardcoded id here?

**Review by rush-skills — APPROVED** [review]
LGTM overall

File: contrib/core/tests/test_action_sendmail.py (hunk 1 above)
Comment (cognifloyd) [reply contrib/core/tests/test_action_sendmail.py:39]: I don't see a problem with creating a mock using hard-coded values.

**Review by khushboobhatia01 — APPROVED** [review]

**Pull request merged**